### ISSUE TITLE
shrink action's info output shows 'Node "{0}" ...' instead of actual node's name

### ISSUE THREAD

**Issue by oododa** [elastic/curator#1142]
shrink action's info output says:

```
2018-02-06 20:54:53,709 INFO      Node "{0}" has multiple data paths and will not be used for shrink operations.
2018-02-06 20:54:54,120 INFO      Node "{0}" has multiple data paths and will not be used for shrink operations.
2018-02-06 20:54:54,492 INFO      Node "{0}" has multiple data paths and will not be used for shrink operations.
2018-02-06 20:54:54,882 INFO      Node "{0}" has multiple data paths and will not be used for shrink operations.
```

This is because curator/actions.py, line 1918 is:

```
                self.loggit.info('Node "{0}" has multiple data paths and will not be used for shrink operations.')
```
It should be:

```
                self.loggit.info('Node "{0}" has multiple data paths and will not be used for shrink operations.'.format(name))
```

**Comment by dustin-decker** [comment]
Looks like I missed this one in https://github.com/elastic/curator/pull/1063

**Comment by untergeek** [comment]
Closed by #1148
